pkg/api/v1alpha1/snowmachineconfig_types.go

```diff
@@ -38,6 +42,9 @@ type SnowMachineConfigSpec struct {
 
 	// Devices contains a device ip list assigned by the user to provision machines.
 	Devices []string `json:"devices,omitempty"`
+
+	// ContainersVolume provides the configuration options for the containers data storage volumes.
+	ContainersVolume *snowv1.Volume `json:"containersVolume,omitempty"`
```

Comment:
we just have to be careful with this so we don't update our API by accident when updating the dependency on the snow module

Maybe it would be wise to define our own volume struct

Reply:
We chatted offline. Will keep the volume type from upstream. Whenever there is API change upstream, it should be a breaking change and we will handle it on our end too.